packages/frontend/core/src/components/root-app-sidebar/invite-members-button.tsx

```diff
@@ -0,0 +1,52 @@
+import { MenuItem } from '@affine/core/modules/app-sidebar/views';
+import { WorkspaceSubscriptionService } from '@affine/core/modules/cloud';
+import { WorkspaceDialogService } from '@affine/core/modules/dialogs';
+import { WorkspacePermissionService } from '@affine/core/modules/permissions';
+import { WorkspaceService } from '@affine/core/modules/workspace';
+import { useI18n } from '@affine/i18n';
+import { CollaborationIcon } from '@blocksuite/icons/rc';
+import { useLiveData, useService } from '@toeverything/infra';
+import { useCallback } from 'react';
+
+import { useWorkspaceInfo } from '../hooks/use-workspace-info';
+
+export const InviteMembersButton = () => {
+  const workspace = useService(WorkspaceService).workspace;
+  const subscription = useService(WorkspaceSubscriptionService).subscription;
+  const permissionService = useService(WorkspacePermissionService);
+
+  const isTeam = useWorkspaceInfo(workspace.meta)?.isTeam;
+  const workspaceSubscription = useLiveData(subscription.subscription$);
+
+  const isOwner = useLiveData(permissionService.permission.isOwner$);
+  const isAdmin = useLiveData(permissionService.permission.isAdmin$);
+  const isLocal = workspace.flavour === 'local';
+  const isOwnerOrAdmin = isOwner || isAdmin;
+
+  const dialogService = useService(WorkspaceDialogService);
+  const onOpenInviteMembersModal = useCallback(() => {
+    dialogService.open('invite-team-member', {
```

Comment:
why not opening `workspace setting > members`?

Reply:
I thought so, but there is no way to trigger invite members dialog after opening the setting/members panel. Let me confirm with the product team first